extensions/Cake.NSwag.yml

```diff
@@ -10,4 +10,5 @@ Categories:
 - swagger
 - api
 - openapi
-TargetCakeVersion: 0.17.0
+TargetCakeVersion: 0.15.2
```

Comment:
This change can be explained by the fact that we are now analyzing the most recent "stable" release:

![image](https://user-images.githubusercontent.com/112710/101060228-2e4d9880-355d-11eb-9c72-79865f49b78b.png)